gymnasium/envs/registration.py

```diff
@@ -147,47 +122,245 @@ class EnvSpec:
     name: str = field(init=False)
     version: int | None = field(init=False)
 
+    # applied wrappers
+    applied_wrappers: tuple[WrapperSpec, ...] = field(init=False, default_factory=tuple)
+
     def __post_init__(self):
-        """Calls after the spec is created to extract the namespace, name and version from the id."""
-        # Initialize namespace, name, version
+        """Calls after the spec is created to extract the namespace, name and version from the environment id."""
         self.namespace, self.name, self.version = parse_env_id(self.id)
 
     def make(self, **kwargs: Any) -> Env:
         """Calls ``make`` using the environment spec and any keyword arguments."""
-        # For compatibility purposes
         return make(self, **kwargs)
 
+    def to_json(self) -> str:
```

Comment:
@pseudo-rnd-thoughts should the `env_spec_dict` be returned in this method?

Reply:
Updated